db/controllers/reviews.controllers.js

```diff
@@ -8,3 +11,20 @@ exports.getReviewById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.incrementVotes = (req, res, next) => {
+  const {
+    params: { review_id: id },
+    body: { inc_votes: votes },
+  } = req;
+  // console.log(typeof votes, ":", votes, "<<<votes");
+  if (!votes) next({ status: 404, msg: "Passed malformed body" });
+  if (typeof votes !== "number")
```

Comment:
this should be in the model and you also have the error handler for checking if the votes exist in the model and controller. It should only be in the model